Makefile

```diff
@@ -19,7 +19,7 @@ test-json:
 	go test -cover -race ./json
 
 test-json-bugs:
-	go test -cover -race ./json/bugs/...
+	go test -race ./json/bugs/...
```

Comment:
Any reason for removing the coverage for this run?

Reply:
Got reply over slack: https://twilio.slack.com/archives/G01CPV0P0SE/p1701104274453339?thread_ts=1701100801.803679&cid=G01CPV0P0SE